### ISSUE TITLE
PHP 8.2 Deprecation: base64_encode(): Passing null to parameter #1 ($string) of type string is deprecated

### ISSUE THREAD

**Issue by chadwells** [verbb/formie#1823]
### Describe the bug

We are seeing this deprecation error show up in our logs after moving to a new host with PHP 8.2.17. Trying to determine if the issue is with Formie, or with something down the trace. But I thought I'd post the issue.

```
yii\base\ErrorException: base64_encode(): Passing null to parameter #1 ($string) of type string is deprecated
#31 /home/forge/www.fireclaytile.com/vendor/verbb/formie/src/elements/Form.php(1768): yii\base\ErrorHandler::handleError
#30 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/web/ErrorHandler.php(79): craft\web\ErrorHandler::handleError
#29 [internal](0): base64_encode
#28 /home/forge/www.fireclaytile.com/vendor/verbb/formie/src/elements/Form.php(1768): verbb\formie\elements\Form::applyRenderOptions
#27 /home/forge/www.fireclaytile.com/vendor/verbb/formie/src/services/Rendering.php(90): verbb\formie\services\Rendering::renderForm
#26 /home/forge/www.fireclaytile.com/vendor/verbb/formie/src/services/Rendering.php(258): verbb\formie\services\Rendering::registerAssets
#25 /home/forge/www.fireclaytile.com/vendor/verbb/formie/src/variables/Formie.php(166): verbb\formie\variables\Formie::registerAssets
#24 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Extension/CoreExtension.php(1635): twig_get_attribute
#23 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/helpers/Template.php(129): craft\helpers\Template::attribute
#22 /home/forge/www.fireclaytile.com/storage/runtime/compiled_templates/89/8969eeef07aa063e142bd00530afba18.php(162): __TwigTemplate_7f50f8f4e947e6b060ebabe81d05b894::doDisplay
#21 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(394): Twig\Template::displayWithErrorHandling
#20 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(367): Twig\Template::display
#19 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(379): Twig\Template::render
#18 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\TemplateWrapper::render
#17 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Extension/CoreExtension.php(1347): twig_include
#16 /home/forge/www.fireclaytile.com/storage/runtime/compiled_templates/f7/f7fb5e58559c47b4294b71de4e5ef8ad.php(317): __TwigTemplate_94ba6ab21624b0581e43a805456bff52::doDisplay
#15 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(394): Twig\Template::displayWithErrorHandling
#14 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(367): Twig\Template::display
#13 /home/forge/www.fireclaytile.com/storage/runtime/compiled_templates/89/899ae5c55bf4ad8c6d53134ec50a4283.php(70): __TwigTemplate_5c1b7a42db4cdc782e13c95f7fea2f84::doDisplay
#12 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(394): Twig\Template::displayWithErrorHandling
#11 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(367): Twig\Template::display
#10 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Template.php(379): Twig\Template::render
#9 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\TemplateWrapper::render
#8 /home/forge/www.fireclaytile.com/vendor/twig/twig/src/Environment.php(280): Twig\Environment::render
#7 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/web/View.php(482): craft\web\View::renderTemplate
#6 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/web/View.php(535): craft\web\View::renderPageTemplate
#5 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\TemplateResponseFormatter::format
#4 /home/forge/www.fireclaytile.com/vendor/yiisoft/yii2/web/Response.php(1100): yii\web\Response::prepare
#3 /home/forge/www.fireclaytile.com/vendor/craftcms/cms/src/web/Response.php(337): craft\web\Response::prepare
#2 /home/forge/www.fireclaytile.com/vendor/yiisoft/yii2/web/Response.php(340): yii\web\Response::send
#1 /home/forge/www.fireclaytile.com/vendor/yiisoft/yii2/base/Application.php(390): yii\base\Application::run
#0 /index.php(12): null
```

### Steps to reproduce

1. 
2. 

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

4.8.2

### Plugin version

2.1.10

### Multi-site?

No

### Additional context

Our environment:
```
PHP version	8.2.17
OS version	Linux 5.15.0-67-generic
Database driver & version	MySQL 8.0.36
Image driver & version	Imagick 3.7.0 (ImageMagick 6.9.11-60)
Craft edition & version	Craft Pro 4.8.2
Yii version	2.0.48.1
Twig version	v3.8.0
Guzzle version	7.8.1

Plugins
Amazon S3	2.2.0
Asset Rev	7.0.0
Back In Stock for Craft Commerce	2.0.0
Blitz	4.10.3
Color Mixer	3.0.0
Cookie Consent Banner	2.0.1
Craft Commerce	4.5.2
Embedded Assets	3.2.0
Environment Label	4.0.3
Feed Me	5.4.0
Field Manager	3.0.8
Flatworld	dev-craft4
Formie	2.1.10
Imager X	4.3.1
Imager X AWS Serverless Transformer	2.1.0
Instant Analytics GA4	4.0.1
Linkit	4.0.4.1
Postie	3.1.5
Redactor	3.0.4
Retcon	2.7.5
Retour	4.1.15
Salesforce	4.0.0
Scout	3.3.3
Sentry Logger	4.1.5
SEOmatic	4.0.43
Spoon	4.0.4
Sprig	2.7.3
Stripe for Craft Commerce	4.1.1
Super Table	3.0.12
Twigpack	4.0.0-beta.4
Typogrify	4.0.1
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.10"`.

**Comment by engram-design** [comment]
Fixed in [2.1.11](https://github.com/verbb/formie/releases/tag/2.1.11)
